third_party/conan/docker/tags.sh

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Maps conan profiles to docker image tags. The images are expected to be named after the conan profile,
+# but the tag can be adjusted here to support versioning of docker containers.
+declare -rA docker_image_tag_mapping=( \
+  [iwyu]="2" \
```

Comment:
From the description, it is a bit unclear to me, why there is only the mapping for iwyu. Could you clarify the comment?

Reply:
Yes, makes sense. I will explain that in the comment.

Reply:
Done.